cep-abi3.md

```diff
@@ -0,0 +1,245 @@
+# Support for abi3 python packages
+
+<table>
+<tr><td> Title </td><td> A new recipe format </td>
```

Comment:
This looks like a leftover from the template

Reply:
```suggestion
<tr><td> Title </td><td> Support for abi3 python packages </td>
```